### PR TITLE
Refs BOT-1156 ~ Now we check if redis response timeout.

### PR DESCRIPTION
**Pull request by Netmisa**
# Description

This PR fix tme out errot  with redis.
Now we call navitia if we have it.

### CODE REVIEW

File: chaos/navitia.py [hunk 1]
```diff
@@ -83,7 +83,13 @@ def _navitia_caller(self, query):
     def get_pt_object(self, uri, object_type, pt_objects=None):
         cache_key = 'Chaos.get_pt_object.{}.{}.{}.{}.{}'.format(self.coverage, self.get_coverage_publication_date(),
                                                                 uri, object_type, pt_objects)
-        cached_pt_object = cache.get(cache_key)
+        try:
+            cached_pt_object = cache.get(cache_key)
+        except:
+            logging.getLogger(__name__).error('Redis time out !')
```
Comment (kinnou02) [inline chaos/navitia.py]:
```suggestion
            logging.getLogger(__name__).exception('Redis time out !')
```

Comment (dvdn) [inline chaos/navitia.py]:
```suggestion
            logging.getLogger(__name__).error('Cache Timeout')
            logging.getLogger(__name__).info('Trying to call Navitia')
```
(pour plus générique si demain on ne passe plus par Redis)

File: chaos/navitia.py [hunk 2]
```diff
@@ -114,8 +120,13 @@ def get_pt_object(self, uri, object_type, pt_objects=None):
...
             if self.collections[object_type] in json and json[self.collections[object_type]]:
                 json_pt_object = json[self.collections[object_type]][0]
-                cache.set(cache_key, json_pt_object,
-                          app.config['CACHE_CONFIGURATION'].get('NAVITIA_CACHE_TIMEOUT', 3600))
+                try:
+                    cache.set(cache_key, json_pt_object,
+                            app.config['CACHE_CONFIGURATION'].get('NAVITIA_CACHE_TIMEOUT', 3600))
+                except:
+                    logging.getLogger(__name__).exception('Cache Timeout')
+                    logging.getLogger(__name__).info('Set value in memory failed.   ')
```
Comment (dvdn) [inline chaos/navitia.py]: spaces
```suggestion
                    logging.getLogger(__name__).info('Set value in memory failed.')
```

File: chaos/navitia.py [hunk 3]
```diff
@@ -83,7 +83,13 @@ def _navitia_caller(self, query):
     def get_pt_object(self, uri, object_type, pt_objects=None):
         cache_key = 'Chaos.get_pt_object.{}.{}.{}.{}.{}'.format(self.coverage, self.get_coverage_publication_date(),
                                                                 uri, object_type, pt_objects)
-        cached_pt_object = cache.get(cache_key)
+        try:
```
Comment (dvdn) [inline chaos/navitia.py]:
```suggestion
      publication_date = 'undefined'
        try:
            publication_date = self.get_coverage_publication_date()
            cache_key = 'Chaos.get_pt_object.{}.{}.{}.{}.{}'.format(self.coverage, publication_date,
                                                                    uri, object_type, pt_objects)
```

**Comment by Netmisa** [comment]
Jenkins could you test this please.

**Pull request merged**